gc/base/standard/Scavenger.hpp

```diff
@@ -588,6 +588,13 @@ class MM_Scavenger : public MM_Collector
 	 * Common merge logic used for both thread and increment level merges.
 	 */
 	void mergeGCStatsBase(MM_EnvironmentBase *env, MM_ScavengerStats *finalGCStats, MM_ScavengerStats *scavStats);
+	/**
+	 * Merge RB increment stats to general increment stats.
+	 * RB occasionally updates separate copy stats fields in the increment stats,
+	 * but eventually those values need to be added to the general copy increment stats.
+	 */
+	void mergeReadBarrierStats(MM_EnvironmentBase *env);
```

Comment:
`env` missing as a `param[in]`. In `Scavenger.hpp`, the methods at the start correctly included `env` as a `param[in]` in the documentation. since `clearIncrementGCStats`, `env` is no longer included as `param[in]` in the method description, which should be fixed.